test/gtest/uct/test_p2p_rma.cc

```diff
@@ -69,7 +69,7 @@ ucs_status_t uct_p2p_rma_test::get_zcopy(uct_ep_h ep, const mapped_buffer &sendb
 void uct_p2p_rma_test::test_xfer(send_func_t send, size_t length,
                                  unsigned flags, uct_memory_type_t mem_type)
 {
-    mapped_buffer sendbuf(length, SEED1, sender(), 1);
+    mapped_buffer sendbuf(length, SEED1, sender(), 1, mem_type);
```

Comment:
to make it compatible with other cuda TLs,  allocate send buffer of mem_type only for ipc.

```
+    uct_memory_type_t src_mem_type = UCT_MD_MEM_TYPE_HOST;
+
+
+    if ((GetParam()->tl_name.compare("cuda_ipc") == 0)) {
+        src_mem_type = mem_type;
+    }
+
+    mapped_buffer sendbuf(length, SEED1, sender(), 1, src_mem_type);

```